src/lib/layer-manager.js

```diff
@@ -51,7 +51,6 @@ export default class LayerManager {
       viewportChanged: true,
       pickingFBO: null
     };
-    this.redrawNeeded = true;
```

Comment:
Yes in principle it looks like it is redundant, although this was originally modeled after redraw flag handling in other objects, where you could call a setRedrawNeeded and force a redraw.

getRedrawNeeded would return true if either redrawNeeded was set, or other flags indicated that redraw was needed, and this would clear redrawNeeded.

The reason there was no returns in the middle of the function was that it was important to clear all redraw flags of subobjects like layers, otherwise they could trigger a redraw on next call.